parm/config/config.defaults.s2sw

```diff
@@ -47,3 +47,6 @@ IOSRV=1
 export ARCH_GAUSSIAN="YES"
 export ARCH_GAUSSIAN_FHMAX=${FHMAX_GFS_00}
 export ARCH_GAUSSIAN_FHINC=${FHOUT_GFS}
+
+# for forecast namelist
+export domains_stack_size="6000000"
```

Comment:
Is this the domain stack size for either with or without aero?  Because when we have aerosols this needs to be much bigger, but if you do not, I think it's fine to use the smaller amount?

Reply:
Only gocart needs it, but I don't think it hurts to use the same without it.

Reply:
But perhaps I misunderstand it's total impact when we add Thompson MP

Reply:
Increased to 16000000 for the time being.